docs/topics/assembly_building-and-deploying-the-booster-to-openshiftlocal.adoc

```diff
@@ -13,7 +13,8 @@ You have two options for executing the {parameter-mission-name} booster locally
 * xref:deploying-the-booster-using-launcher-osl_{context}[Using {launcher}]
 * xref:deploying-the-{parameter-mission}-booster-using-the-oc-cli-client_{context}[Using the `oc` CLI client]
 
-There is no functional difference between these methods, choose the one you prefer.
+While both methods use the same `oc` commands to deploy your application, using
```

Comment:
* Replace _While_ with _Although_ or _Even though_. [IBM Style Guide]
* Suggestion: Put the whole sentence on one line as per [AsciiDoc Recommended Practices](http://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line).